lib/tasks/scan_for_missing_records.rake

```diff
@@ -1,244 +1,214 @@
 desc 'Scan for missing foreign key records'
 namespace :missing_foreign_key_relations do
-    task scan: :environment do
```

Comment:
The first commit is the substantive change; second is updating whitespace from 4 spaces to 2.

Reply:
Ooops, I had not realized at all I used 4 space. 🐍 habits die hard.